src/track/trackmetadatataglib.cpp

```diff
@@ -419,10 +419,10 @@ bool parseSeratoMarkers(
         const QByteArray& data) {
     DEBUG_ASSERT(pTrackMetadata);
 
-    SeratoMarkers seratoMarkers(pTrackMetadata->getTrackInfo().getSeratoMarkers());
-    bool isValid = SeratoMarkers::parse(&seratoMarkers, data);
+    SeratoTags seratoTags(pTrackMetadata->getTrackInfo().getSeratoTags());
+    bool isValid = seratoTags.parseMarkers(data);
```

Comment:
This local variable doesn't make the code more readable

Reply:
This is just mirroring all the other related functions in the file, e.g. https://github.com/mixxxdj/mixxx/blob/master/src/track/trackmetadatataglib.cpp#L410-L414